application/helpers/remotecontrol/remotecontrol_handle.php

```diff
@@ -2072,6 +2072,92 @@ public function invite_participants($sSessionKey, $iSurveyID )
             return array('status' => 'No permission');
     }
 
+    /**
+     * RPC Routine to send register mails to participants in a survey
+     * Returns array of results of sending
+     *
+     * @access public
+     * @param string $sSessionKey Auth credentials
+     * @param int $iSurveyID ID of the survey that participants belong
+     * @param array $overrideAllConditions (optional) replace the default conditions, like this:
+     *   $overrideAllConditions = Array();
+     *   $overrideAllConditions[] = 'tid = 2';
+     *   $response = $myJSONRPCClient->mail_registered_participants( $sessionKey, $survey_id, $overrideAllConditions );
+     * @return array Result of the action
+     */
+    public function mail_registered_participants($sSessionKey, $iSurveyID, $overrideAllConditions=Array() )
+    {
+        Yii::app()->loadHelper('admin/token');
+        if (!$this->_checkSessionKey($sSessionKey))
+            return array('status' => 'Invalid session key');
+
+        $oSurvey = Survey::model()->findByPk($iSurveyID);
+        if (!isset($oSurvey))
+            return array('status' => 'Error: Invalid survey ID');
+
+        if (Permission::model()->hasSurveyPermission($iSurveyID, 'tokens', 'update'))
+        {
+
+            if(!tableExists("{{tokens_$iSurveyID}}"))
+                return array('status' => 'Error: No token table');
+
+			$command = new CDbCriteria();
```

Comment:
LS is not consistent in this respect. $criteria, $condition(s) and $command are all in use. I chose for $command, because this is what findUninvited() and findUninvitedIDs() in application/models/TokenDynamic.php use. If we ever want to change invite_participants and add $overrideAllConditions there as well, we could copy/paste parts of TokenDynamic.php to the RC handler.